content/news/playtest-20160403.md

```diff
@@ -0,0 +1,118 @@
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &ndash; from 7 to 11 &ndash; during this development cycle!
+
+We will cover these changes in more detail in the news post for the third playtest. In the meantime, the only thing you really need to know is that the game won't upgrade maps automatically anymore. Please download new revisions from [the OpenRA Resource Center](http://resource.openra.net) instead.
+
+<div style="text-align:center" markdown="1">
+![Upgrading maps in the resource center](/images/news/20160403-web-resource-center-map-upgrade.png)
+<p>
+Use the Resource Center to upgrade maps easily to new versions of OpenRA.
+</p>
+</div>
+
+Support for third-party mods will significantly improve with the next release. It is now possible to create an archive of your whole mod (called ".oramod"). Installing it on other computers is then just a matter of placing that archive in the "mods" subdirectory of the Support folder.
+
+The news post for the second playtest will detail what changes your mod needs to make it compatible with the upcoming release and to enable the .oramod packaging. All shipped mods have already been converted, so feel free to take a look at those.
+
+In the future, OpenRA will feature an ingame mod browser that will allow you to search for and install mods directly from the Resource Center.
+
+<hr />
+## Dedicated server changes
+<hr />
+Dedicated servers have seen quite a few improvements during this cycle, which make them altogether much easier to run.
+
+<div style="font-family: monospace; border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<em>
+$ ./OpenRA.Server.exe<br />
+Starting dedicated server for mod: ra<br />
+[2016-04-02T16:06:11] Master server communication established.<br />
+[2016-04-02T16:10:56] Player has joined the game.<br />
+[2016-04-02T16:11:49] Newbie has joined the game.<br />
+[2016-04-02T16:12:16] Antares changed the map to Doughnut.<br />
+Game started<br />
+[2016-04-02T16:37:38] Player has disconnected.<br />
+[2016-04-02T16:37:47] Antares has disconnected.<br />
+[2016-04-02T16:37:54] Newbie has disconnected.<br />
+No one is playing, shutting down...<br />
+Starting a new server instance...<br />
+[2016-04-02T16:37:55] Master server communication established.<br />
+</em>
+</div>
+
+First and foremost, the biggest change is that dedicated servers no longer need to sync maps with the Resource Center. Especially when you were running multiple servers of different versions, this could quickly become a headache. This will also mean that the distinction between synced and unsynced servers will be gone. That's right: all maps will be available on all servers, immediately after upload to the Resource Center, without any further action required. 
```

Comment:
This is more of a "but I would have done it that way" than a complaint, so feel free to ignore it, but this may be clearer as a bullet point list.